src/tracks/SampleTrack.cpp

```diff
@@ -826,6 +836,19 @@ SampleTrackView::SampleTrackView( SampleTrack * _t, TrackContainerView* tcv ) :
 	m_panningKnob->setLabel( tr( "PAN" ) );
 	m_panningKnob->show();
 
+	m_activityIndicator = new FadeButton(QApplication::palette().color(QPalette::Active,
+							QPalette::Background),
+						QApplication::palette().color(QPalette::Active,
+							QPalette::BrightText),
+						QApplication::palette().color(QPalette::Active,
+							QPalette::BrightText).darker(),
+						getTrackSettingsWidget());
```

Comment:
```suggestion
	m_activityIndicator = new FadeButton(
		QApplication::palette().color(QPalette::Active, QPalette::Background),
		QApplication::palette().color(QPalette::Active, QPalette::BrightText),
		QApplication::palette().color(QPalette::Active, QPalette::BrightText).darker(),
		getTrackSettingsWidget()
	);
```

I feel like this is easier to read.